app/controllers/admin/users_controller.rb

```diff
@@ -0,0 +1,94 @@
+class Admin::UsersController < ApplicationController
+  before_action :authenticate_admin!
+
+  before_action :find_user, only: [:edit, :destroy, :update, :deactivate, :activate]
+
+  helper_method :resource
+
+  def index
+    @users = User.reorder('created_at DESC').page(params[:page])
+
+    respond_to do |format|
+      format.html
+      format.json { render json: @users }
+    end
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    admin = params[:user].delete(:admin)
+    @user = User.new(params[:user])
```

Comment:
That would mean we need to remove `attr_accessible` from the user model, not sure about the implications. Would it be enough to configure `devise_parameter_sanitizer` for everything else to work?